<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValoracionForm</class>
 <widget class="QGroupBox" name="ValoracionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>90</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Valoración Historico Artistica</string>
  </property>
  <property name="title">
   <string>Valoración Historico Artistica</string>
  </property>
  <widget class="QLineEdit" name="cuentaEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>-10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="valEstiloLbl">
      <property name="minimumSize">
       <size>
        <width>68</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>68</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Estilo:</string>
      </property>
      <property name="buddy">
       <cstring>valEstiloEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="valEstiloEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maxLength">
       <number>80</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="valCalificacionLbl">
      <property name="minimumSize">
       <size>
        <width>68</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>68</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Calificación:</string>
      </property>
      <property name="buddy">
       <cstring>valCalificacionCmb</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="valCalificacionCmb">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Ninguno</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
